defaults/main.yml

```diff
@@ -17,60 +17,45 @@ walle_script_location: /usr/local/bin/walle.py
 walle_filesize_min: 0
 walle_filesize_max: 10
 walle_since_hours: 24
-walle_envs_database:
-  - key: MALWARE_LIB
-    value: "{{ walle_malware_database_location }}/{{ walle_database_file }}"
-  - key: PGPASSFILE
-    value: "{{ walle_pgpass_file }}"
-  - key: GALAXY_CONFIG_FILE
-    value: "{{ galaxy_config_dir }}/galaxy.yml"
-  - key: GALAXY_LOG_DIR
-    value: "{{ galaxy_log_dir }}"
-  - key: PGHOST
-    value: 127.0.0.1
-  - key: PGUSER
-    value: galaxy
-  - key: PGDATABASE
-    value: galaxy
-  - key: GALAXY_PULSAR_APP_CONF
-    value: "{{ galaxy_config_dir }}/pulsar_app.yml"
-  - key: GXADMIN_PATH
-    value: /usr/local/bin/gxadmin
-
-# These will be added to the default env vars - you can override
-# walle_envs_database by defining the same key in walle_extra_env_vars
-walle_extra_env_vars: []
+walle_verbose: false
+walle_kill: false
+# walle_tool: <string to filter tool ids in database job table>
 
-walle_env_vars: "{{ walle_envs_database + walle_extra_env_vars }}"
+walle_envs_database:
+  MALWARE_LIB: "{{ walle_malware_database_location }}/{{ walle_database_file }}"
+  PGPASSFILE: "{{ walle_pgpass_file }}"
+  GALAXY_CONFIG_FILE: "{{ galaxy_config_dir }}/galaxy.yml"
+  GALAXY_LOG_DIR: "{{ galaxy_log_dir }}"
+  PGHOST: 127.0.0.1
+  PGUSER: galaxy
+  PGDATABASE: galaxy
+  GALAXY_PULSAR_APP_CONF: "{{ galaxy_config_dir }}/pulsar_app.yml"
+  GXADMIN_PATH: /usr/local/bin/gxadmin
 
 # delete users when malware was found and malware severity reached walle_delete_threshold
+walle_api_key: null  # admin api key to delete users, goes to VAULT
```

Comment:
They're not appended to the bashrc when `null`, because they are removed by the `combine` filter, right?

Reply:
Good question - they are not removed but rendered in `walle_bashrc` like:

```
export GALAXY_API_KEY=""
export GALAXY_BASE_URL=""
```

I think the result in walle.py is the same as if they were unset:

```py
        try:
            if len(from_env := os.environ.get(env, "").strip()) == 0:
                raise ValueError
            else:
                return from_env
        except ValueError:
            logger.error(f"Path for %s is invalid", env)
            raise ValueError
```

Are you happy with that?

Reply:
Yes, sorry for being over-accurate

Reply:
Not at all - I appreciate your eyes! :D